shared/style/dialer/dialer.css

```diff
@@ -1,5 +1,12 @@
+* {
+  /* By default do not allow any user text selection, only enable for certain
+  parts on a per-element/region area */
+  -moz-user-select: none;
```

Comment:
Hi Doug,

I did some more testing on this and what i found was that I needed to set it so none for the *.  If I set it to none on the *, then it's not needed on the html or body.  Also, adding -moz-select-user: none to both the html, and the body (without adding it at the *) is not sufficient to keep unwanted elements from being selected.